swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, category='Clothing', condition=0, age=0):
+        super().__init__(category, condition, age)
```

Comment:
This works great! We might consider removing the optional `category` argument here. We know we want the the category always to be `"Clothing"`, so we can hardcode that into the `super()__init__` call instead of asking the caller of the `Clothing` constructor to supply it. Ditto for the other `Item` subclasses.